prow/pjutil/pjutil.go

```diff
@@ -171,33 +171,37 @@ func kubeEnv(environment map[string]string) []kube.EnvVar {
 	return kubeEnvironment
 }
 
-// PartitionPending separates the provided prowjobs into pending and non-pending
+// PartitionActive separates the provided prowjobs into pending and triggered
```

Comment:
Please mention that completed prowjobs are filtered out.

Reply:
done